src/main/java/fi/metatavu/kuntaapi/server/rest/ptv7adapter/Ptv7Adapter.java

```diff
@@ -0,0 +1,260 @@
+    return translateEntity(ElectronicServiceChannel.class, cloneEntity(ElectronicServiceChannel.class, entity), getMappings().getElectoricServiceChannel());
+  }
+
+  /**
+   * Translates PTV8 to use PTV7 naming conventions
+   * 
+   * @param entity entity
+   * @return translated entity
+   * @throws IOException thrown on reading error
+   */
+  public PrintableFormServiceChannel translate(PrintableFormServiceChannel entity) throws IOException {
+    return translateEntity(PrintableFormServiceChannel.class, cloneEntity(PrintableFormServiceChannel.class, entity), getMappings().getPrintableFormServiceChannel());
+  }
+
+  /**
+   * Translates PTV8 to use PTV7 naming conventions
+   * 
+   * @param entity entity
+   * @return translated entity
+   * @throws IOException thrown on reading error
+   */
+  public ServiceLocationServiceChannel translate(ServiceLocationServiceChannel entity) throws IOException {
+    return translateEntity(ServiceLocationServiceChannel.class, cloneEntity(ServiceLocationServiceChannel.class, entity), getMappings().getServiceLocationServiceChannel());
+  }
+
+  /**
+   * Translates PTV8 to use PTV7 naming conventions
+   * 
+   * @param entity entity
+   * @return translated entity
+   * @throws IOException thrown on reading error
+   */
+  public PhoneServiceChannel translate(PhoneServiceChannel entity) throws IOException {
+    return translateEntity(PhoneServiceChannel.class, cloneEntity(PhoneServiceChannel.class, entity), getMappings().getPhoneServiceChannel());
+  }
+
+  /**
+   * Translates PTV8 to use PTV7 naming conventions
+   * 
+   * @param entity entity
+   * @return translated entity
+   * @throws IOException thrown on reading error
+   */
+  public WebPageServiceChannel translate(WebPageServiceChannel entity) throws IOException {
+    return translateEntity(WebPageServiceChannel.class, cloneEntity(WebPageServiceChannel.class, entity), getMappings().getWebPageServiceChannel());
+  }
+  
+  /**
+   * Translates PTV8 to use PTV7 naming conventions
+   * 
+   * @param entity entity
+   * @return translated entity
+   * @throws IOException thrown on reading error
+   */
+  public Service translate(Service entity) throws IOException {
+    return translateEntity(Service.class, cloneEntity(Service.class, entity), getMappings().getService());
+  }
+  
+  /**
+   * Applies PTV8 -> PTV7 naming convention into an object 
+   * 
+   * @param entityClass entity class
+   * @param entity entity
+   * @param entityMapping entity mapping
+   * @return cloned entity using PTV7 naming convention
+   * @throws IOException thrown on reading error
+   */
+  private <T> T translateEntity(Class<T> entityClass, T entity, Map<String, Map<String, String>> entityMapping) throws IOException {
+    if (entity == null) {
+      return null;
+    }
+    
+    T result = cloneEntity(entityClass, entity);
+    
+    entityMapping.keySet().forEach((propertyPath) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "propertyPath" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)